frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -51,8 +53,14 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
       direction={{ default: 'column' }}
       grow={{ default: 'grow' }}
       className="ocs-getting-started-card"
+      data-test-id={`ocs-getting-started-card-${id}`}
     >
-      <Title headingLevel="h3" size={TitleSizes.md} style={{ color: titleColor }}>
+      <Title
+        headingLevel="h3"
+        size={TitleSizes.md}
+        style={{ color: titleColor }}
+        data-test-id="ocs-getting-started-card-title"
```

Comment:
Here we don't really need fully qualified IDs because we should be first selecting the card before selecting any of its children.
We should discuss this as part of the conventions with the team.

My first thought is that this could simply be `data-test-id="title"`